src/content/changes/highlights/2023-09-14.md

```diff
@@ -0,0 +1,15 @@
+---
+date: 2023-09-14T14:00:00
+title: Highlights for the week ending September 14 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Management API
+- We introduced the new custom resource `RoleBindingTemplate` to all management clusters. It allows dynamic creation and deletion of RoleBindings across organizations. Read the [docs]({{< relref "use-the-api/management-api/authorization/#role-binding-templates" >}}) for more information.
```

Comment:
I saw this reference format in another highlight but I'm not sure it's still correct :thinking: should it be a direct link?